_includes/search-lunr.html

```diff
@@ -0,0 +1,71 @@
+<script src="/js/lunr.js"></script>
```

Comment:
If we use the js file from our repository, what would be the best way to manage versions and updates to future versions of this and other js libs we'll use?

Reply:
to be honest, I have no idea.  If we vendor the script in the repo, do we ever need to change it?  (Sorry if this is a dumb question)

Reply:
Sorry let me re-phrase my question whats the best alternative?  What other options do we have?